policy/modules/system/systemd.te

```diff
@@ -801,6 +806,8 @@ optional_policy(`
 	dbus_connect_system_bus(systemd_networkd_t)
 	dbus_watch_system_bus_runtime_dirs(systemd_networkd_t)
 	dbus_watch_system_bus_runtime_named_sockets(systemd_networkd_t)
+	dbus_read_system_bus_runtime_dirs(systemd_networkd_t)
+	dbus_read_system_bus_runtime_named_sockets(systemd_networkd_t)
```

Comment:
Thinking about these further it may make more sense to add these to `dbus_system_bus_client` instead.

Reply:
I added those two calls directly in dbus_system_bus_client.